(dp1
S'output'
p2
S'Unable to open included view file: /home/ayush/Desktop/web2py/applications/UIpart/views/layou2.html'
p3
sS'layer'
p4
S'ParserContainer'
p5
sS'code'
p6
S'{{extend \'layou2.html\'}}\n\n<h2>{{=T( request.args(0).replace(\'_\',\' \').capitalize() )}}</h2>\n<div id="web2py_user_form">\n{{\nif request.args(0)==\'login\':\n    if not \'register\' in auth.settings.actions_disabled:\n        form.add_button(T(\'Register\'),URL(args=\'register\', vars={\'_next\': request.vars._next} if request.vars._next else None),_class=\'btn\')\n    pass\n    if not \'request_reset_password\' in auth.settings.actions_disabled:\n        form.add_button(T(\'Lost Password\'),URL(args=\'request_reset_password\'),_class=\'btn\')\n    pass\npass\n=form\n}}\n</div>\n<script language="javascript"><!--\njQuery("#web2py_user_form input:visible:enabled:first").focus();\n{{if request.args(0)==\'register\':}}\n    web2py_validate_entropy(jQuery(\'#auth_user_password\'),100);\n{{elif request.args(0)==\'change_password\':}}\n    web2py_validate_entropy(jQuery(\'#no_table_new_password\'),100);\n{{pass}}\n//--></script>\n\n'
p7
sS'snapshot'
p8
(dp9
S'exception'
p10
(dp11
S'__getslice__'
p12
S"<method-wrapper '__getslice__' of exceptions.IOError object>"
p13
sS'__str__'
p14
S"<method-wrapper '__str__' of exceptions.IOError object>"
p15
sS'__reduce__'
p16
S'<built-in method __reduce__ of exceptions.IOError object>'
p17
sS'__dict__'
p18
S'{}'
p19
sS'__sizeof__'
p20
S'<built-in method __sizeof__ of exceptions.IOError object>'
p21
sS'__init__'
p22
S"<method-wrapper '__init__' of exceptions.IOError object>"
p23
sS'__setattr__'
p24
S"<method-wrapper '__setattr__' of exceptions.IOError object>"
p25
sS'__reduce_ex__'
p26
S'<built-in method __reduce_ex__ of exceptions.IOError object>'
p27
sS'__new__'
p28
S'<built-in method __new__ of type object>'
p29
sS'errno'
p30
S'2'
sS'__format__'
p31
S'<built-in method __format__ of exceptions.IOError object>'
p32
sS'__class__'
p33
S"<type 'exceptions.IOError'>"
p34
sS'filename'
p35
S"'/home/ayush/Desktop/web2py/applications/UIpart/views/layou2.html'"
p36
sS'__doc__'
p37
S"'I/O operation failed.'"
p38
sS'__getitem__'
p39
S"<method-wrapper '__getitem__' of exceptions.IOError object>"
p40
sS'__setstate__'
p41
S'<built-in method __setstate__ of exceptions.IOError object>'
p42
sS'__getattribute__'
p43
S"<method-wrapper '__getattribute__' of exceptions.IOError object>"
p44
sS'args'
p45
S"(2, 'No such file or directory')"
p46
sS'__subclasshook__'
p47
S'<built-in method __subclasshook__ of type object>'
p48
sS'__unicode__'
p49
S'<built-in method __unicode__ of exceptions.IOError object>'
p50
sS'strerror'
p51
S"'No such file or directory'"
p52
sS'__delattr__'
p53
S"<method-wrapper '__delattr__' of exceptions.IOError object>"
p54
sS'__repr__'
p55
S"<method-wrapper '__repr__' of exceptions.IOError object>"
p56
sS'__hash__'
p57
S"<method-wrapper '__hash__' of exceptions.IOError object>"
p58
ssS'evalue'
p59
S"[Errno 2] No such file or directory: '/home/ayush/Desktop/web2py/applications/UIpart/views/layou2.html'"
p60
sS'frames'
p61
(lp62
(dp63
S'file'
p64
S'/home/ayush/Desktop/web2py/gluon/template.py'
p65
sS'dump'
p66
(dp67
S'builtinopen'
p68
S'<built-in function open>'
p69
sS'fileobj'
p70
S'undefined'
p71
sS'filepath'
p72
S"'/home/ayush/Desktop/web2py/applications/UIpart/views/layou2.html'"
p73
ssS'lnum'
p74
I447
sS'lines'
p75
(dp76
I448
S'            text = fileobj.read()'
p77
sI449
S'            fileobj.close()'
p78
sI450
S'        except IOError:'
p79
sI451
S"            self._raise_error('Unable to open included view file: ' + filepath)"
p80
sI442
S'        # Get the path of the file on the system.'
p81
sI443
S'        filepath = self.path and os.path.join(self.path, filename) or filename'
p82
sI444
S''
sI445
S'        # try to read the text.'
p83
sI446
S'        try:'
p84
sI447
S"            fileobj = open(filepath, 'rb')"
p85
ssS'call'
p86
S"(self=<gluon.template.TemplateParser object>, filename='layou2.html')"
p87
sS'func'
p88
S'_get_file_text'
p89
sasS'pyver'
p90
S'Python 2.7.5+: /usr/bin/python (prefix: /usr)'
p91
sS'etype'
p92
S"<type 'exceptions.IOError'>"
p93
sS'date'
p94
S'Sun Dec  1 15:06:21 2013'
p95
sS'locals'
p96
(dp97
S'self'
p98
S'<gluon.template.TemplateParser object>'
p99
sg35
S"'layou2.html'"
p100
sS'context'
p101
S"{'A': <class 'gluon.html.A'>, 'ANY_OF': <class 'gluon.validators.ANY_OF'>, 'Auth': <class 'gluon.tools.Auth'>, 'B': <class 'gluon.html.B'>, 'BEAUTIFY': <class 'gluon.html.BEAUTIFY'>, 'BODY': <class 'gluon.html.BODY'>, 'BR': <class 'gluon.html.BR'>, 'BUTTON': <class 'gluon.html.BUTTON'>, 'CAT': <class 'gluon.html.CAT'>, 'CENTER': <class 'gluon.html.CENTER'>, ...}"
p102
sS'filepath'
p103
S"'/home/ayush/Desktop/web2py/applications/UIpart/views/layou2.html'"
p104
sssS'traceback'
p105
S'Traceback (most recent call last):\n  File "/home/ayush/Desktop/web2py/gluon/template.py", line 447, in _get_file_text\n    fileobj = open(filepath, \'rb\')\nIOError: [Errno 2] No such file or directory: \'/home/ayush/Desktop/web2py/applications/UIpart/views/layou2.html\'\n'
p106
s.